spring-cloud-task-samples/multiple-datasources/src/main/resources/application.properties

```diff
@@ -1,2 +1,4 @@
 spring.application.name=Demo Multiple DataSources Task
 logging.level.org.springframework.cloud.task=DEBUG
+
+spring.profiles.default=embedded
```

Comment:
IntelliJ is showing a warning here: `Cannot resolve configuration property 'spring.profiles.default' `. Should this be `spring.profiles.active=embedded` as in https://github.com/spring-cloud/spring-cloud-task/pull/727/files#diff-18da3e6827339e29aef887fbb5946d77bb4b38966bd0f636c83b0b37937a3a2fR38 ?

Reply:
Good catch.  Thanks!